app/services/atmosphere/vmt_migrator.rb

```diff
@@ -0,0 +1,40 @@
+require 'migratio/worker/openstack_amazon_migrator'
+require 'migratio/worker/openstack_openstack_migrator'
+
+module Atmosphere
+  class VmtMigrator
+    def initialize(virtual_machine_template,
+                   source_compute_site, destination_compute_site)
+      @virtual_machine_template = virtual_machine_template
+      @source_compute_site = source_compute_site
+      @destination_compute_site = destination_compute_site
+    end
+
+    def select_migrator
+      case @destination_compute_site.technology
+      when 'aws'
+        Migratio::Worker::OpenstackAmazonMigrator
+      when 'openstack'
+        Migratio::Worker::OpenstackOpenstackMigrator
+      end
```

Comment:
Consider renaming method to select_migrator_class or return an instance of migrator instead of its class.